tools/kvmexit.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python
+#
+# kvmexit.py
+#
+# Display the exit_reason and its statistics of each kvm_exit
+# for all vcpus of all virtual machines. For example,
+# $./kvmexit.py
+#  TGID     PID      KVM_EXIT_REASON                     STAT
+#  1273551  1273568  EXIT_REASON_MSR_WRITE               6
+#  1274253  1274261  EXIT_REASON_EXTERNAL_INTERRUPT      1
+#  1274253  1274261  EXIT_REASON_HLT                     12
+#  ...
+#
+# @TGID: each vitual machine's pid in the user space.
+# @PID: the user space's thread of each vcpu of that virtual machine.
+# @KVM_EXIT_REASON: the reason why the vm exits.
+# @STAT: the counts of the @KVM_EXIT_REASON since the vm starts, and
+#        it is recorded every second by setting `sleep(1)`.
```

Comment:
@COUNT: the counts of the @KVM_EXIT_REASONS.

"since vm starts" is not accurate. the counting starts when bpf starts to run. The "sleep(1)" is not true either.